### ISSUE TITLE
hx-trigger from: with css attribute equality selector

### ISSUE THREAD

**Issue by nathanahtan** [bigskysoftware/htmx#782]
I'm trying to use hx-trigger with the from: modifier to select nodes with the css attribute equality selector (like:  div[attribute="attribute_value"], and am getting syntax errors.

Should this work?:

`hx-trigger="from:input[data-hx-id=&quot;368b8acd-41d6-48da-a518-9ef78e218b77&quot;]"`

Thanks.

**Comment by geoffrey-eisenbarth** [comment]
Not sure about your use of `&quot;`, but I think you need to specify an event (change, load, etc). 

On my end, I had a similar use case and I was able to get the following to work: `hx-trigger="change from:[name^=foo]"`

**Comment by nathanahtan** [comment]
The `&quot;` was injected by the browser when I copied it out.  In the source template, they were nested `"`:

`hx-trigger="from:input[data-hx-id="368b8acd-41d6-48da-a518-9ef78e218b77"]"`

I thought I tried without the inner quotes as well, but I'll try again, as that is the only difference I see.  I'm not sure I specified the changed event either, figuring that was the default for an input tag.

Thanks.
